Null pointer exception in ArrayList initialization in concrete

**Description**

Try to generate unit tests with standard flow for `spring-boot-testing` project, method `OrderService.getOrders`.


**Expected behavior**

Tests are generated properly. 
There are no `Default concrete execution failed messages`. 

**Actual behavior**

The following exceptions is found in logs:

```java
Caused by: java.lang.NullPointerException: Cannot read the array length because "a" is null
	at java.base/java.util.ArrayList.<init>(ArrayList.java:182)
	at org.utbot.instrumentation.instrumentation.execution.constructors.UtModelConstructor.findEqualValueOfWellKnownType(UtModelConstructor.kt:352)
	at org.utbot.instrumentation.instrumentation.execution.constructors.UtModelConstructor.constructFromAny(UtModelConstructor.kt:341)
	at org.utbot.instrumentation.instrumentation.execution.constructors.UtModelConstructor.construct(UtModelConstructor.kt:178)
	at org.utbot.instrumentation.instrumentation.execution.constructors.UtModelConstructor.construct(UtModelConstructor.kt:137)
	at org.utbot.instrumentation.instrumentation.execution.phases.ModelConstructionPhase.convertToExecutionResult(ModelConstructionPhase.kt:121)
	at org.utbot.instrumentation.instrumentation.execution.SimpleUtExecutionInstrumentation$invoke$1$1$2.invoke(SimpleUtExecutionInstrumentation.kt:106)
	at org.utbot.instrumentation.instrumentation.execution.SimpleUtExecutionInstrumentation$invoke$1$1$2.invoke(SimpleUtExecutionInstrumentation.kt:92)
	at org.utbot.instrumentation.instrumentation.execution.phases.PhasesController$executePhaseInTimeout$1$result$1.invoke(PhasesController.kt:78)
	at org.utbot.common.ThreadBasedExecutor$invokeWithTimeout$1.invoke(ThreadUtil.kt:70)
	at org.utbot.common.ThreadBasedExecutor$ensureThreadIsAlive$1.invoke(ThreadUtil.kt:124)
	at org.utbot.common.ThreadBasedExecutor$ensureThreadIsAlive$1.invoke(ThreadUtil.kt:120)
	at kotlin.concurrent.ThreadsKt$thread$thread$1.run(Thread.kt:30)

```

--- a/utbot-instrumentation/src/main/kotlin/org/utbot/instrumentation/instrumentation/execution/constructors/UtModelConstructor.kt
+++ b/utbot-instrumentation/src/main/kotlin/org/utbot/instrumentation/instrumentation/execution/constructors/UtModelConstructor.kt
@@ -348,12 +348,14 @@ class UtModelConstructor(
                 ?: constructCompositeModel(value, remainingDepth)
         }
 
-    private fun findEqualValueOfWellKnownType(value: Any): Pair<Any, ClassId>? = when (value) {
-        is List<*> -> ArrayList(value) to listClassId
-        is Set<*> -> LinkedHashSet(value) to setClassId
-        is Map<*, *> -> LinkedHashMap(value) to mapClassId
-        else -> null
-    }
+    private fun findEqualValueOfWellKnownType(value: Any): Pair<Any, ClassId>? = runCatching {
+        when (value) {
+            is List<*> -> ArrayList(value) to listClassId
+            is Set<*> -> LinkedHashSet(value) to setClassId
+            is Map<*, *> -> LinkedHashMap(value) to mapClassId
+            else -> null
+        }
+    }.getOrNull()
 
     /**
      * Constructs custom UtModel but does it only for predefined list of classes.

